freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -127,6 +129,14 @@ def _handle_auto_rebuild(self, db_event):
             "Orignial nvrs of build in the advisory #{0} are: {1}".format(
                 self.event.advisory.errata_id, " ".join(original_nvrs)))
 
+        # Ignore builds from dependent events that were rebuilt recently
+        for orig_nvr, rebuilt_nvr in nvrs_mapping.copy().items():
+            done_build = db_event.get_artifact_build_from_event_dependencies(rebuilt_nvr)
```

Comment:
This is a little confusing, the `nvrs_mapping` is a mapping of images in advisory and their original image NVRs, they're operator or operand images, and the images to skip (`event.container_images`) is the bundle images in the result of images to be rebuilt in this handler, could you correct me if I'm wrong? I believe the skipping feature is only possible to be done after the rebuilt list is figured out and before recording them in db.